docs/migrating/migratingfrommvc5/migratingfrommvc5.rst

```diff
@@ -0,0 +1,258 @@
+
+.. code-block:: c#
+
+	"dependencies": {
+		"Microsoft.AspNet.Server.IIS": "1.0.0-beta3",
+		"Microsoft.AspNet.Mvc": "6.0.0-beta3"
+	},
+
+Now open Startup.cs and modify it as follows:
+
+.. code-block:: c#
+
+	public void ConfigureServices(IServiceCollection services)
+	{
+		services.AddMvc();
+	}
+
+	public void Configure(IApplicationBuilder app)
+	{
+		app.UseMvc(routes =>
+		{
+			routes.MapRoute(
+				name: "default",
+				template: "{controller}/{action}/{id?}",
+				defaults: new { controller = "Home", action = "Index" });
+		});
+	}
+
+At this point we are ready to create a simple Controller and View. Add a Controllers folder and a Views folder to the project. Add an MVC Controller called HomeController.cs class to the Controllers folder and a Home folder in the Views folder. Finally, add an Index.cshtml MVC View Page to the Views/Home folder. The project structure should be as shown:
+
+.. image:: _static/project-structure-controller-view.png
+
+Modify Index.cshtml to show a welcome message:
+
+.. code-block:: html
+
+	<h1>Hello world!</h1>
+
+Run the application - you should see Hello World output in your browser.
+
+.. image:: _static/hello-world.png
+
+Migrate Basic Controllers, Views, and Static Content
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now that we've confirmed we have a simple, working ASP.NET MVC 6 project, it's time to start migrating functionality from the source project. There are many different ways one can approach this task. We will need to move all of the client-side content files (CSS, fonts, scripts), all of the controllers, views, and models, and migrate configured features like bundling, filters, and identity. Let's begin by replacing our simple "hello world" implementation of HomeController with the actual HomeController and Views from the source project.
+
+Copy each of the methods from the source HomeController to the HomeController we added to the project in the previous section. Note that in MVC 5, actions typically returned ActionResult, but in MVC 6 this has changed to IActionResult (though it will still compile if you leave it as ActionResult).
+
+Next, create new MVC View Pages in the Views/Home folder for About and Contact. Copy the contents of the corresponding views in the old project to these new views, as well as Index.cshtml. At this point you should once again be able to run the new application, and although the styles are not yet in place, you should see the correct content on the home page as well as /home/about and /home/contact (contact is shown here):
```

Comment:
Need to cover what to do with web.config in Views folder. Convert to _GlobalImport.cshtml with `@using` statements.